nilearn/input_data/nifti_labels_masker.py

```diff
@@ -49,8 +50,17 @@ class NiftiLabelsMasker(BaseMasker, CacheMixin):
         See http://nilearn.github.io/manipulating_images/input_output.html
         Region definitions, as one image of labels.
 
+    labels : list of str, optional
+        Full labels corresponding to the labels image. This is used
+        to improve reporting quality if provided.
+        Warning: The labels must be consistent with the label
```

Comment:
But we need to check that actively.

Reply:
So far I am checking that the number of provided string labels matches the number of integer labels in the label image, but I don't see a way to check that they weren't shuffled.

Reply:
Sorry I had missed that. LGTM